inlang/source-code/paraglide/paraglide-js/src/utils/git.ts

```diff
@@ -15,11 +15,12 @@ export async function getGitRemotes(
 ) {
 	try {
 		const usedFs = args.nodeishFs || fs
-		const root = await findRoot({ fs: usedFs, filepath: args.filepath || process.cwd() })
```

Comment:
getGitOrigins seems not to be used because its currently commented out - is this on purpose? 
@jannesblobel @LorisSigrist 
https://github.com/opral/monorepo/blame/bbe6c2cea7ce3d4446d01487520218cfb0390569/inlang/source-code/paraglide/paraglide-js/src/services/telemetry/implementation.ts#L5

Reply:
Before we had projectIDs we were using the git-origin to differentiate between projects. We can remove this now

Reply:
@martin-lysk see https://linear.app/opral/issue/LIX-6/move-parseorigin-from-inlangtelemetry-to-lix-jsclient